src/analysis/analyzer.ts

```diff
@@ -1,4 +1,4 @@
-import fs, {readFileSync, statSync} from "fs";
+import sysfs from "fs";
```

Comment:
I've not had any luck tracking down where / how you determine what `sysfs` is within Node's `fs` package.

On a meta point, how are you navigating node's built-in packages like this? And where can I look over the node `fs` interface?